### PR TITLE
Update the Link of config.json in README

### LINKED ISSUE #2528: Dead link in README
There is a dead link in the README pointing to https://github.com/exercism/docs/blob/master/language-tracks/configuration/exercises.md
image

### PR DESCRIPTION
**Pull request by mintyknight**
This fixes #2528
The link of the config file wad dead.

### CODE REVIEW

**Comment by mintyknight** [comment]
Hi, @andrerfcsantos, do you mind helping review? This was coded according to your suggestions in #2528.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Let's walk through an example, non-existent, exercise, which we'll call
 
 ### Exercise configuration
 
-An exercise is [configured](https://github.com/exercism/docs/blob/master/language-tracks/configuration/exercises.md)
+An exercise is [configured](https://exercism.org/docs/building/tracks/config-json)
```
Comment (junedev) [inline README.md:49]: I would recommend to remove the link here entirely. It doesn't really add value when the link is provided below with clear text that this is where to read more.

**Review by junedev — APPROVED** [review]
One minor idea for improvement, otherwise looks correct.

File: README.md (hunk 1 above)
Comment (mintyknight) [reply README.md]: Done, such link removed

**Review by junedev — APPROVED** [review]

**Pull request merged**